File: fizzbuzz.js
var multThree;
var multFive;
var multOf;

multThree = function(x){
  return x % 3 === 0;
};

multFive = function(x){
  return x % 5 === 0;
}

multOfEval = function(x){
  if(multThree(x) && multFive(x)){
    return "FizzBuzz";
  } else if (multThree(x)){
    return "Fizz";
  } else if (multFive(x)){
    return "Buzz";
  } else {
    return x;
  }
}

File: fizzbuzz.rb
def mult_three(i)
  i % 3 == 0
end

def mult_five(i)
  i % 5 == 0
end

def mult_of(i)
  if mult_three(i) && mult_five(i)
    "FizzBuzz"
  elsif mult_three(i)
    "Fizz"
  elsif mult_five(i)
    "Buzz"
  else
    i
  end
end

integers = Array(1..100)
integers.map!{|i| mult_of(i)}

#For if printing each value is necessary
integers.each{|i| p i}